targets/TARGET_STM/TARGET_STM32H7/TARGET_STM32H747xI/system_clock.c

```diff
@@ -160,7 +160,7 @@ MBED_WEAK uint8_t SetSysClock_PLL_HSE(uint8_t bypass)
 /******************************************************************************/
 /*            PLL (clocked by HSI) used as System clock source                */
 /******************************************************************************/
-uint8_t SetSysClock_PLL_HSI(void)
+MBED_WEAK uint8_t SetSysClock_PLL_HSI(void)
```

Comment:
You should not have to update this function.
I can see the difference is HAL_PWREx_ConfigSupply call, I will think about this...

Reply:
Yup, we need a way to override the power supply scheme and it was the easiest one